api/rest/client/methods.go

```diff
@@ -77,9 +77,9 @@ func (c *Client) Unpin(ci *cid.Cid) error {
 
 // Allocations returns the consensus state listing all tracked items and
 // the peers that should be pinning them.
-func (c *Client) Allocations() ([]api.Pin, error) {
+func (c *Client) Allocations(hidden bool) ([]api.Pin, error) {
```

Comment:
I'm not much into this naming. `all`, `hidden`, `a`... if we are going to filter, have a filter type int as receiver with values for `all/meta/pin/cdag/shard`.

`Allocations(pinType)`

where -1 means all.